packages/rn-tester/RNTesterIntegrationTests/RCTLoggingTests.m

```diff
@@ -90,7 +90,7 @@ - (void)testLogging
 
   XCTAssertEqual(_lastLogLevel, RCTLogLevelError);
   XCTAssertEqual(_lastLogSource, RCTLogSourceJavaScript);
-  XCTAssertEqualObjects(_lastLogMessage, @"Invariant Violation: Invariant failed");
+  XCTAssertTrue([_lastLogMessage containsString:@"Invariant Violation: Invariant failed"]);
```

Comment:
`js engine: Hermes` is added to the end of the error message / invariant when Hermes is on. Replaced with `containsString` to make it more universal.

CC: @alloy